apps/charge-accounts-service/src/operators/operators.service.ts

```diff
@@ -98,35 +327,42 @@ export class OperatorsService {
     }
   }
 
-  async handleWebhook (webhookEvent: WebhookEvent): Promise<{ address: string, valueEth: string } | undefined> {
-    this.logger.log(`handleWebhook: ${JSON.stringify(webhookEvent)}`)
-    if (
-      webhookEvent.direction !== 'incoming' ||
-      webhookEvent.tokenType !== 'FUSE'
-    ) {
-      return
-    }
-
-    return {
-      address: webhookEvent.to.toLowerCase(),
-      valueEth: webhookEvent.valueEth
-    }
+  async checkWalletActivationStatus (auth0Id) {
+    const user = await this.usersService.findOneByAuth0Id(auth0Id)
+    const wallet = await this.findWalletOwner(user._id)
+    return wallet?.isActivated || false
   }
 
-  async addWebhookAddress (params: { walletAddress: string, webhookId: string, apiKey: string }) {
+  async addAddressToOperatorsWebhook (walletAddress: string) {
     const apiBaseUrl = this.configService.get('CHARGE_BASE_URL')
-    const url = `${apiBaseUrl}/api/v0/notifications/webhook/add-addresses?apiKey=${params.apiKey}`
+    const apiKey = this.configService.get('PAYMASTER_FUNDER_API_KEY')
+    const webhookId = this.configService.get('PAYMASTER_FUNDER_WEBHOOK_ID')
+    const url = `${apiBaseUrl}/api/v0/notifications/webhook/add-addresses?apiKey=${apiKey}`
     const requestBody = {
-      webhookId: params.webhookId,
-      addresses: [params.walletAddress]
+      webhookId,
+      addresses: [walletAddress]
+    }
+    try {
+      return await this.httpProxyPost(url, requestBody)
```

Comment:
@NikolayRn are we still going to use httpProxyPost? As you mentioned in the task:

> We should to use proper microservice interaction way, for example we can add/get webhooks via CallMsFunction instead of making requests to our own backend via API service.

Reply:
Yep, forgot about it, thank you